CHANGELOG.md

```diff
@@ -37,7 +37,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   "Required Annotation: Owner/Team", "Required Label: Owner/Team" and "Latest tag"
 - ROX-21207: All usage of RocksDB has been removed.
   - Support to directly upgrade from any ACS version 3.74.x or earlier to version 4.5 has been deprecated. ACS version 3.74.x or earlier MUST be upgraded to version 4.4.x before proceeding to version 4.5 or later.
-
+- ROX-23714: Populating initial registry integration repo list (`/v2/_catalog`) will now be done lazily on first call to `Match`.
```

Comment:
I think you can leave out the implementation detail of `Match`. May be confusing because that's internal

Reply:
https://github.com/stackrox/stackrox/pull/11186/commits/8b8ad991e290224d423a05ab23575fa59c6f4df4